tools/mat_setup_workspace.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+
+"Install angular source dependencies"
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
+
+def mat_setup_workspace():
+    """This repository rule should be called from your WORKSPACE file.
+
+    It creates some additional Bazel external repositories that are used internally
+    to build angular material
+    """
+    # Use Bazel managed node modules. See more below: 
+    # https://github.com/bazelbuild/rules_nodejs#bazel-managed-vs-self-managed-dependencies
+    yarn_install(
+        name = "matdeps",
+        package_json = "@angular_material//:package.json",
+        yarn_lock = "@angular_material//:yarn.lock",
+    )
```

Comment:
Can we use two spaces for indentation in the whole file? We have it like that in all other build files.

<sup>I know, most of the Bazel projects use tab, but I think since we have all other files with _two_, we should be consistent_</sup>

Reply:
Sure. Its 4 spaces for many bazel projects but 2 spaces is consistent for this repo. Should add `buildifier` at some point.

Reply:
Yeah we'll do at some point.